filament/backend/src/vulkan/VulkanPipelineCache.h

```diff
@@ -184,6 +186,25 @@ class VulkanPipelineCache : public CommandBufferObserver {
 private:
     static constexpr uint32_t ALL_COMMAND_BUFFERS = (1 << VK_MAX_COMMAND_BUFFERS) - 1;
 
+    struct LayoutBundleKey {
+        // Currently, only samplers can be bound to each shader stages.
+        std::array<VkShaderStageFlags, SAMPLER_BINDING_COUNT> samplers;
+    };
```

Comment:
Note that this entire key could fit inside 64 bits. (2 bits per sampler * 32 samplers).

Reply:
Yes it's possible. LayoutBundleKey's type is changed to bitset64.